<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSubMainWidget</class>
 <widget class="QWidget" name="KSubMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <widget class="QPushButton" name="latexPushButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWebView" name="latexWebView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
